recipes/cimg/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class CImgConan(ConanFile):
+    name = "cimg"
+    description = "The CImg Library is a small and open-source C++ toolkit for image processing"
+    homepage = "http://cimg.eu"
+    topics = "conan", "cimg", "physics", "simulation", "robotics", "kinematics", "engine",
+    license = "CeCILL V2"
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    settings = "compiler"
+
+    _cimg_options = [
+        ("enable_fftw",         "cimg_use_fftw",       "fftw/3.3.8"),
+        ("enable_jpeg",         "cimg_use_jpeg",       "libjpeg/9d"),
+        ("enable_openexr",      "cimg_use_openexr",    "openexr/2.4.0"),
+        ("enable_png",          "cimg_use_png",        "libpng/1.6.37"),
+        ("enable_tiff",         "cimg_use_tiff",       "libtiff/4.0.9"),
```

Comment:
Maybe a TODO note here? So you can see when building the package as a warning message

Reply:
What do you mean?
Warning the user that cimg has these options enabled?

Reply:
Including `# TODO` in your code, Conan will print automatically that message when building the package.

Reply:
I con't find the TODO in the logs, created by CI

Reply:
I see, because that's was a feature when pylint was embedded in Conan. Now it's a separated hook: https://github.com/conan-io/hooks/blob/master/hooks/recipe_linter.py